frontend/src/lib/components/DatabaseTableTree/TreeRow.scss

```diff
@@ -1,4 +1,7 @@
 .TreeRow {
+    display: flex;
```

Comment:
Ahh, we fixed the same thing here lol. https://github.com/PostHog/posthog/pull/21293 but yours goes one step further